restic/internal/restore_test.py

```diff
@@ -38,8 +38,19 @@ def test_restore_specific_snapshot_id_and_include(self, mock_execute):
 
     @mock.patch.object(restore.command_executor, 'execute')
     def test_restore_specific_snapshot_id_and_exclude(self, mock_execute):
```

Comment:
Following from the other comment, can we leave this test's call as-is but change the assertion to be that it throws the right exception?

Reply:
Done, I am unsure about how to use `mock_execute` so I just set it to empty string (to avoid unused variable warning).